sdk/metricsadvisor/azure-ai-metricsadvisor/samples/async_samples/sample_credential_entities_async.py

```diff
@@ -0,0 +1,152 @@
+            connection_string=connection_string,
+            data_source_credential_description="my credential entity",
+        )
+    )
+
+    return credential_entity
+    # [END create_credential_entity_async]
+
+
+async def sample_get_credential_entity_async(credential_entity_id):
+    # [START get_credential_entity_async]
+    from azure.ai.metricsadvisor import MetricsAdvisorKeyCredential, MetricsAdvisorAdministrationClient
+
+    service_endpoint = os.getenv("METRICS_ADVISOR_ENDPOINT")
+    subscription_key = os.getenv("METRICS_ADVISOR_SUBSCRIPTION_KEY")
+    api_key = os.getenv("METRICS_ADVISOR_API_KEY")
+
+    client = MetricsAdvisorAdministrationClient(service_endpoint,
+                                  MetricsAdvisorKeyCredential(subscription_key, api_key))
+
+    credential_entity = client.get_credential_entity(credential_entity_id)
+
+    print("Credential entity type: {}".format(credential_entity.data_source_credential_type))
+    print("Credential entity name: {}".format(credential_entity.data_source_credential_name))
+    print("Description: {}".format(credential_entity.data_source_credential_description))
+
+    # [END get_credential_entity_async]
+
+
+async def sample_list_credential_entities_async():
+    # [START list_credential_entities_async]
+    from azure.ai.metricsadvisor import MetricsAdvisorKeyCredential, MetricsAdvisorAdministrationClient
+
+    service_endpoint = os.getenv("METRICS_ADVISOR_ENDPOINT")
+    subscription_key = os.getenv("METRICS_ADVISOR_SUBSCRIPTION_KEY")
+    api_key = os.getenv("METRICS_ADVISOR_API_KEY")
+
+    client = MetricsAdvisorAdministrationClient(service_endpoint,
+                                  MetricsAdvisorKeyCredential(subscription_key, api_key))
+
+    credential_entities = client.list_credential_entities()
+    for credential_entity in credential_entities:
+        print("Credential entity type: {}".format(credential_entity.data_source_credential_type))
+        print("Credential entity name: {}".format(credential_entity.data_source_credential_name))
+        print("Description: {}\n".format(credential_entity.data_source_credential_description))
+
+    # [END list_credential_entities_async]
+
+
+async def sample_update_credential_entity_async(credential_entity):
+    # [START update_credential_entity_async]
+    from azure.ai.metricsadvisor import MetricsAdvisorKeyCredential, MetricsAdvisorAdministrationClient
+
+    service_endpoint = os.getenv("METRICS_ADVISOR_ENDPOINT")
+    subscription_key = os.getenv("METRICS_ADVISOR_SUBSCRIPTION_KEY")
+    api_key = os.getenv("METRICS_ADVISOR_API_KEY")
+
+    client = MetricsAdvisorAdministrationClient(service_endpoint,
+                                  MetricsAdvisorKeyCredential(subscription_key, api_key))
+
+    credential_entity.description = "updated description"
+
+    client.update_credential_entity(credential_entity)
+    updated = client.get_credential_entity(credential_entity.data_source_credential_id)
```

Comment:
I thought that update_credential_entity returned the updated credential entity. Do we need this get?